apps/meteor/tests/end-to-end/api/02-channels.js

```diff
@@ -1938,6 +1938,62 @@ describe('[Channels]', function () {
 			});
 		});
 
+		it(`should successfully convert a channel to a team when the channel's name is sent as parameter`, (done) => {
+			updatePermission('create-team', ['admin']).then(() => {
+				updatePermission('edit-room', ['admin']).then(() => {
```

Comment:
there is no need to update permissions.. looking the previous test, this is the current state already.